source/vibe/http/websockets.d

```diff
@@ -386,9 +386,9 @@ final class WebSocket {
 		On the JavaScript side, the text will be available as message.data (type string).
 		Throws: WebSocketException if the connection is closed.
 	*/
-	void send(string data)
+	void send(const(char)[] data)
```

Comment:
It's at least a documentation cue, and I *really*  hope that it gets implemented one day, or at least that more illegal cases are detected (but it actually needs to be implemented correctly so that `@safe` isn't broken).